src/app/lib/analyticsUtils/frontpage.js

```diff
@@ -0,0 +1,36 @@
+import deepGet from '../utilities/deepGet';
+
+export const getPageIdentifier = frontpageData =>
+  deepGet(['metadata', 'analyticsLabels', 'counterName'], frontpageData) ||
+  'unknown.page';
+
+const guidRegex =
+  '([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})';
+const curieRegex = new RegExp(
+  `http://www.bbc.co.uk/asset/${guidRegex}/desktop/domestic`,
+);
+export const getContentId = frontpageData => {
```

Comment:
Can you fix the spacing in the file please? Ideally a empty line above and below `if` statements please